src/workScheduleDays/types.ts

```diff
@@ -12,4 +12,6 @@ export type WorkScheduleDay = {
   weekday: string;
 };
 
-export type WorkScheduleDayUpdate = Omit<Partial<WorkScheduleDay>, 'timesheet_template_id' | 'weekday_id'>;
+export type WorkScheduleDayUpdate = Omit<Partial<WorkScheduleDay>, 'timesheet_template_id' | 'weekday_id'> & {
+  id: number;
+};
```

Comment:
Maybe 
```suggestion
export type WorkScheduleDayUpdate = Pick<WorkScheduleDay, 'id'> & Omit<Partial<WorkScheduleDay>, 'timesheet_template_id' | 'weekday_id' | 'id'>;
```
For the unlikely case that our id field's type changes, so that we don't need to hunt down the re-definition :shrug: